.github_changelog_generator

```diff
@@ -0,0 +1,10 @@
+issues=false
```

Comment:
I didn't know github changelog generator 👍

Reply:
It's kinda good _(but the documentation is awful)_